README.md

```diff
@@ -1,3 +1,10 @@
+# Alpha Site
+
+[![Build Status](https://teamcity.dts-stn.com/app/rest/builds/buildType:(id:Dev_AlphaSite_Build)/statusIcon)](https://teamcity.dts-stn.com/viewType.html?buildTypeId=Dev_AlphaSite_Build&guest=1)
+
+
```

Comment:
can we also add a badge for the link to the application

Reply:
k i will update that.